R/zzz.R

```diff
@@ -12,22 +12,23 @@ namespaces_reset <- function() {
 
 namespaces_default <- function() {
   has_glue <- requireNamespace("glue", quietly = TRUE)
+  is_running_gha <- isTRUE(Sys.getenv("GITHUB_ACTIONS") == "true") && !needs_stdout()
 
   list(
     global = list(
       default = list(
         threshold = as.loglevel(Sys.getenv("LOGGER_LOG_LEVEL", unset = "INFO")),
-        layout    = layout_simple,
+        layout    = if (is_running_gha) layout_gha else layout_simple,
         formatter = if (has_glue) formatter_glue else formatter_sprintf,
-        appender  = if (needs_stdout()) appender_stdout else appender_console
+        appender  = if (is_running_gha) appender_stderr else if (needs_stdout()) appender_stdout else appender_console
```

Comment:
I'm not sure if this is needed, as `appender_console` is an alias for `appender_stderr`

Reply:
Ah yeah, you are right. That does make it look way nicer :-)